embedded-graphics/src/geometry/point.rs

```diff
@@ -78,11 +78,66 @@ impl Point {
         Point { x, y }
     }
 
+    /// Creates a point with X and Y values set to an equal value.
+    ///
+    /// ```rust
+    /// use embedded_graphics::geometry::Point;
+    ///
+    /// let filled = Point::new_filled(11);
+    ///
+    /// assert_eq!(filled, Point { x: 11, y: 11 });
+    /// ```
+    pub const fn new_filled(fill: i32) -> Self {
+        Point { x: fill, y: fill }
+    }
+
     /// Creates a point with X and Y equal to zero.
     pub const fn zero() -> Self {
         Point { x: 0, y: 0 }
     }
 
+    /// Return the `Point` with `y` set to `0`.
```

Comment:
IMO the comments for the `x_axis` and `y_axis` methods should also include what happens to the other value.
```suggestion
    /// Returns a point with equal `x` value and `y` set to `0`.
```